--- ganeti-instance-image-0.6/common.sh.in~	2013-07-01 21:04:13.000000000 +0200
+++ ganeti-instance-image-0.6/common.sh.in	2014-02-22 20:53:09.813718196 +0100
@@ -288,7 +288,7 @@
         log_error "Cannot interpret kpartx output and get partition mapping"
         exit 1
     fi
-    $KPARTX -a -p- $blockdev > /dev/null
+    $KPARTX -a -s -p- $blockdev > /dev/null
     filesystem_dev="/dev/mapper/${filesystem_dev_base/%-1/}"
     if [ ! -b "/dev/mapper/$filesystem_dev_base" ]; then
         log_error "Can't find kpartx mapped partition: /dev/mapper/$filesystem_dev_base"
